mod-circulation/src/main/resources/vega/mod-circulation/features/samples/request/move-request-entity-request.json

```diff
@@ -0,0 +1,4 @@
+{
+  "destinationItemId": "#(destinationItemId)",
+  "requestType": "#(requestType)"
+}
```

Comment:
add empty line

Reply:
Also there is another file 'request-entity-request'. What do you about moving it to the same package '../samples/request/'  for consistency ?

Reply:
yes, it can be moved, but then we will be needed to update lots of tests

Reply:
Yes, these are side effects of proper decision.